backend/app/Actions/Visits/CreateVisitAction.php

```diff
@@ -88,6 +88,9 @@ public function execute(CreateVisitRequest $request): void
         $visit->page_id = $page->id;
         $visit->visitor_id = $visitor->id;
         $visit->geo_position_id = $geoPosition->id;
+        $visit->page_load_time = $request->getPageLoadTime();
+        $visit->server_response_time = $request->getServerResponseTime();
+        $visit->domain_lookup_time = $request->getDomainLookupTime();
```

Comment:
For server response time and dns lookup add methods, and make requests in order to calculate this data.